lib/workers/repository/process/index.spec.ts

```diff
@@ -29,6 +29,7 @@ describe('workers/repository/process/index', () => {
     it('processes baseBranches', async () => {
       extract.mockResolvedValue({} as never);
       config.baseBranches = ['branch1', 'branch2'];
+      config.multipleBaseBranches = true; // for coverage
```

Comment:
With this, we're ensuring that the code gets test coverage, but we're not verifying that the covered code actually works. Would it not make sense to take the extra step and verify that too?